src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -165,99 +166,99 @@
   onDestroy(() => ($error = ''));
 </script>
 
-<div class="pb-10">
+<div class="flex flex-col gap-4 pb-10">
   {#if $loading}
     <Loading title={loadingText} />
   {:else}
-    <header class="mb-12 flex flex-col gap-4">
-      <Link href={backHref} icon="chevron-left">
-        {backTitle}
-      </Link>
-      <h1>{title}</h1>
-    </header>
-    <form class="mb-4 flex w-full flex-col gap-4 md:w-2/3 xl:w-1/2">
-      <div class="w-full">
-        <Input
-          id="name"
-          bind:value={name}
-          label={translate('schedules.name-label')}
-          error={errors['name']}
-          maxLength={232}
-          disabled={Boolean(scheduleId)}
-          on:input={onInput}
-          on:blur={onBlur}
-          required
-        />
-      </div>
-      <div class="w-full">
-        <Input
-          id="workflowType"
-          bind:value={workflowType}
-          label={translate('schedules.workflow-type-label')}
-          error={errors['workflowType']}
-          on:input={onInput}
-          on:blur={onBlur}
-          required
-        />
-      </div>
-      <div class="w-full">
-        <Input
-          id="workflowId"
-          bind:value={workflowId}
-          label={translate('schedules.workflow-id-label')}
-          error={errors['workflowId']}
-          on:input={onInput}
-          on:blur={onBlur}
-          required
+    <Link href={backHref} icon="chevron-left">
+      {backTitle}
+    </Link>
+    <h1>{title}</h1>
+    <Card class="w-full xl:w-3/4 2xl:w-1/2">
+      <form class="mb-4 flex w-full flex-col gap-4">
+        <div class="w-full">
```

Comment:
Wondering if we need this div and the similar ones on lines 192, 203, and 214?

Reply:
Doesn't look like we do... [removed](https://github.com/temporalio/ui/pull/2465/commits/dd25bd950a5c1bad985146c193d06880184dfb33) 🗑️